modules/aruco/include/opencv2/aruco.hpp

```diff
@@ -294,7 +294,7 @@ class CV_EXPORTS_W Board {
     CV_WRAP void setIds(InputArray ids);
 
     /// array of object points of all the marker corners in the board
-    /// each marker include its 4 corners in CCW order. For M markers, the size is Mx4.
+    /// each marker include its 4 corners in CW order. For M markers, the size is Mx4.
```

Comment:
It doesn't make sense to mention CW or CCW without specifying used coordinate system.

Please take a look on calib3d module documentation.

Reply:
Added this definition:
```
    /// each marker include its 4 corners in this order:
    ///-   objPoints[i][0] - left-top point of i-th marker
    ///-   objPoints[i][1] - right-top point of i-th marker
    ///-   objPoints[i][2] - right-bottom point of i-th marker
    ///-   objPoints[i][3] - left-bottom point of i-th marker
```